File: backend/HRplatform/HRplatform.Domain/Candidate.cs
namespace HRplatform.Domain
{
    public class Candidate
    {
        public string Id { get; set; } = "";  //GUID
        public string FullName { get; set; } = "";
        public DateOnly DateOfBirth { get; set; }
        public string Email { get; set; } = "";
        public string ContactNum { get; set; } = "";
        public List<Skill> Skills { get; set; } = new List<Skill>();
    }
}

File: frontend/HRplatform-frontend/src/models/Candidate.ts
import type { Skill } from "./Skill";
export interface Candidate {
    id: string;
    fullName: string;
    dateOfBirth: string; 
    email: string;
    contactNum: string;
    skills?: Skill[]; 
}
export interface CreateUpdateCandidateRequest {
    fullName: string;
    dateOfBirth: string | null;
    email: string;
    contactNum: string;
}